doc/developer/conf.py

```diff
@@ -395,8 +395,11 @@ def setup(app):
     # printfrr extensions
     app.add_object_type("frrfmt", "frrfmt", parse_node=parse_frrfmt)
 
-    # css overrides for HTML theme
-    app.add_stylesheet("overrides.css")
+    if "add_css_file" in dir(app):
```

Comment:
Haven't seen this before. I think `hasattr(app, "add_css_file")` is the more accepted way to do this.